include/roboteam_ai/analysis/PlaysObjects/Invariants/AlwaysFalseInvariant.h

```diff
@@ -0,0 +1,31 @@
+//
+// Created by jessevw on 11.12.19.
+//
+
+#ifndef RTT_ALWAYSFALSEINVARIANT_H
+#define RTT_ALWAYSFALSEINVARIANT_H
+
+#include "analysis/PlaysObjects/Invariants/Invariant.h"
+
+namespace rtt::ai::analysis {
+    /**
+     * Invariant that is true when the ball belongs to us
+     */
+    class AlwaysFalseInvariant : Invariant{
+    private:
+        std::string name;
```

Comment:
Shadows a field in the inherited class, rip another sizeof(std::string) bytes (i know that's not much but still)